src/inspect_ai/util/_sandbox/limits.py

```diff
@@ -0,0 +1,74 @@
+from pathlib import Path
+
+from inspect_ai._util.text import truncate_string_to_bytes
+from inspect_ai.util._subprocess import ExecResult
+
+
+class SandboxEnvironmentLimits:
+    """Encapsulates limits for sandbox environments."""
+
+    MAX_EXEC_OUTPUT_SIZE = 1024**2
+    MAX_EXEC_OUTPUT_SIZE_STR = "1 MiB"
+
+    MAX_READ_FILE_SIZE = 100 * 1024**2
+    MAX_READ_FILE_SIZE_STR = "100 MiB"
+
+
+class SandboxOutputLimitExceededError(Exception):
```

Comment:
`SandboxExecOutputLimitExceededError` would be more precise but I didn't want to make this even more wordy.